src/twisted/python/test/test_release.py

```diff
@@ -912,15 +877,26 @@ def test_export(self):
         self.assertStructure(exportDir, structure)
 
 
+try:
+    gitVersion = runCommand(["git", "--version"]).split(b" ")[2].split(b".")
+except FileNotFoundError:
+    gitSkip, gitSkipText = True, "git is not present."
+else:
+    # We want git 2.0 or above.
+    if int(gitVersion[0]) >= 2:
+        gitSkip, gitSkipText = False, ""
+    else:
+        gitSkip, gitSkipText = True, "old git is present"
```

Comment:
I think that in these days all the supported environment if they have git, will have a git older than v2


And maybe we can also assume that all testing system have git... so we can always run it.

I think that this might be a reason why the tests were skipped on Windows, as getting the git path on Windows might be a little bit more complicated.

Reply:
One reason it failed on Windows was that `/tmp` was hardcoded as the temp dir. That's easy to fix though.

Windows using "git-bash" (msys) is actually close enough to a POSIX environment for a lot of tasks. But I don't know if our CI runs under bash or `cmd.exe`.

Reply:
After fixing the temp dir, more issues are popping up. So I disabled the git tests on Windows.

Once the pydoctor and sphinx tests are running in CI, we can see whether those do run correctly on Windows or should be disabled as well.